app/services/IdentityService.scala

```diff
@@ -258,12 +281,21 @@ object IdentityApiClient extends IdentityApiClient with LazyLogging {
       .withWSFailureLogging(endpoint)
       .withCloudwatchMonitoringOfPost
   }
+
+  override val consentEmail: Email => Future[WSResponse] = { email =>
+    val endpoint = authoriseCall(WS.url(s"$identityEndpoint/consent-email"))
+
+    val json = Json.obj("email" -> email, "set-consents" -> List("supporter"))
+    endpoint.post(json)
+      .withWSFailureLogging(endpoint)
+      .withCloudwatchMonitoringOfPost
```

Comment:
I know you're just following the convention here, but are any of these Cloudwatch metrics providing any value?

Reply:
Good question I would defer to you or @paulbrown1982 on that.

Reply:
Personally I'd be inclined to delete them all, rather than adding more. 

I don't think anyone looks at them, and I've had a quick skim over our AWS alarms and there don't seem to be any related to these metrics.

Reply:
People only ever look at them when clutching at straws when not being able to resolve an issue via log lines.

Reply:
It sounds like we should probably remove them then but I think that should be a separate PR